v4l2.go

```diff
@@ -130,7 +137,23 @@ type v4l2_frmsize_stepwise struct {
 	Step_height uint32
 }
 
-//Hack to make go compiler properly align union
+type v4l2_frmivalenum struct {
```

Comment:
can you leave code comments as to where the structs are sourced from (some absolute href to a linux doc page)?

Reply:
Added links to Linux kernel doc ref.

Reply:
We could just add a link at the top of this file to the header file from the kernel docs rather than a separate link for each struct https://www.kernel.org/doc/html/latest/userspace-api/media/v4l/videodev.html#videodev2-h. That link has everything.

Reply:
Yeah, I don't think it's a good idea to copy the docs from the kernel. If you're using this file you're going to refer to the kernel docs anyways. At best we should just link to file we're copying from which I linked in my last comment. Leaving the comments out seem to be standard here anyways and I think for the reasons above.

Reply:
+1

Reply:
Switched to topline comment to ref header file `videodev2.h`.